src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,15 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+interface Props {
+  user: User,
+}
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const { name, email } = user;
+
+  return (
+    <a className="UserInfo" href={`mailto:${email}`}>
```

Comment:
Consider adding a title attribute to the anchor tag to provide more context to users on hover.